app/controllers/admin/users_controller.rb

```diff
@@ -33,7 +33,7 @@ def show
 
     def update
       @user = User.find(params[:id])
-      manage_credits
+      Users::ManageCredits.call(@user, user_params)
```

Comment:
Since this handles user and org credits, I'd prefer `Credits::Manage` as a name.